src/blender/scripts/blinds.py

```diff
@@ -1,84 +1,85 @@
-#	along with OpenShot Video Editor.  If not, see <http://www.gnu.org/licenses/>.
+# You should have received a copy of the GNU General Public License
+# along with OpenShot Video Editor.  If not, see <http://www.gnu.org/licenses/>.
 
 
 # Import Blender's python API.  This only works when the script is being
 # run from the context of Blender.  Blender contains it's own version of Python
 # with this library pre-installed.
 import bpy
 
-# Load a font
+
 def load_font(font_path):
-	""" Load a new TTF font into Blender, and return the font object """
-	# get the original list of fonts (before we add a new one)
-	original_fonts = bpy.data.fonts.keys()
-	
-	# load new font
-	bpy.ops.font.open(filepath=font_path)
-	
-	# get the new list of fonts (after we added a new one)
-	for font_name in bpy.data.fonts.keys():
-		if font_name not in original_fonts:
-			return bpy.data.fonts[font_name]
-		
-	# no new font was added
-	return None
+    """ Load a new TTF font into Blender, and return the font object """
+    # get the original list of fonts (before we add a new one)
+    original_fonts = bpy.data.fonts.keys()
+
+    # load new font
+    bpy.ops.font.open(filepath=font_path)
+
+    # get the new list of fonts (after we added a new one)
+    for font_name in bpy.data.fonts.keys():
+        if font_name not in original_fonts:
+            return bpy.data.fonts[font_name]
+
+    # no new font was added
+    return None
 
 # Debug Info:
 # ./blender -b test.blend -P demo.py
 # -b = background mode
 # -P = run a Python script within the context of the project file
 
+
 # Init all of the variables needed by this script.  Because Blender executes
 # this script, OpenShot will inject a dictionary of the required parameters
 # before this script is executed.
-params = {		
-			'title' : 'Oh Yeah! OpenShot!',
-			'extrude' : 0.1,
-			'bevel_depth' : 0.02,
-			'spacemode' : 'CENTER',
-			'text_size' : 1.5,
-			'width' : 1.0,
-			'fontname' : 'Bfont',
-			
-			'color' : [0.8,0.8,0.8],
-			'alpha' : 1.0,
-			
-			'output_path' : '/tmp/',
-			'fps' : 24,
-			'quality' : 90,
-			'file_format' : 'PNG',
-			'color_mode' : 'RGBA',
-			'horizon_color' : [0.57, 0.57, 0.57],
-			'resolution_x' : 1920,
-			'resolution_y' : 1080,
-			'resolution_percentage' : 100,
-			'start_frame' : 20,
-			'end_frame' : 25,
-			'animation' : True,
-		}
-
-#INJECT_PARAMS_HERE
+params = {
+    'title': 'Oh Yeah! OpenShot!',
+    'extrude': 0.1,
+    'bevel_depth': 0.02,
+    'spacemode': 'CENTER',
+    'text_size': 1.5,
+    'width': 1.0,
+    'fontname': 'Bfont',
+
+    'color': [0.8, 0.8, 0.8],
+    'alpha': 1.0,
+
+    'output_path': '/tmp/',
+    'fps': 24,
+    'quality': 90,
+    'file_format': 'PNG',
+    'color_mode': 'RGBA',
+    'horizon_color': [0.57, 0.57, 0.57],
+    'resolution_x': 1920,
+    'resolution_y': 1080,
+    'resolution_percentage': 100,
+    'start_frame': 20,
+    'end_frame': 25,
+    'animation': True,
+}
+
+# INJECT_PARAMS_HERE
```

Comment:
Whoops! Well, messed this one up. I didn't catch that my reformatting inserted a space here, which breaks this line of the Blender code:

https://github.com/OpenShot/openshot-qt/blob/211eed62a24fd4cc3b5fa55dd3ba02fbed697416/src/windows/views/blender_listview.py#L532-L533

I'll merge a fix in a moment.